pkgs/build-support/rust/default.nix

```diff
@@ -89,6 +81,13 @@ in stdenv.mkDerivation (args // {
     ''}
     EOF
     cat .cargo/config
```

Comment:
I wonder why have this cat actually :)

Reply:
This cat comes exist in `configurePhase` since 912dca1.

Reply:
I would take it, so please remove it from the expression.

Reply:
removed :)